### DISCUSSION TITLE
Where does Foliate store settings?

### DISCUSSION THREAD

**Discussion by in-in** [johnfactotum/foliate#1226] [Q&A] [answered]
gsettings is empty
```sh
gsettings list-recursively com.github.johnfactotum.Foliate
No such schema “com.github.johnfactotum.Foliate”
```

foliate:
ID: com.github.johnfactotum.Foliate
Ref: app/com.github.johnfactotum.Foliate/x86_64/stable
Arch: x86_64
Branch: stable
Version: 3.1.0
License: GPL-3.0-or-later
Origin: flathub
Collection: org.flathub.Stable
Installation: system
Installed: 3.4MB
Runtime: org.gnome.Platform/x86_64/45
Sdk: org.gnome.Sdk/x86_64/45

System: Linux Mint 21.2 Cinnamon

**Comment by johnfactotum** [comment, marked as answer]
For Flatpak, the settings are stored in `~/.var/app/com.github.johnfactotum.Foliate/config/glib-2.0/settings/keyfile`.
